incident/models/incident_index_page.py

```diff
@@ -200,12 +201,16 @@ def get_context(self, request, *args, **kwargs):
             ).only('category')
 
         # Added: .only('category') to both queryset queries above
-
         if incident_filter.cleaned_data:
+            export_filter_data = copy.deepcopy(incident_filter.cleaned_data)
+            for name, value in export_filter_data.items():
+                if isinstance(value, ManyRelationValue) and value.pks:
```

Comment:
I think that technically this does not cover all valid configurations that a ManyRelationValue could have, for instance using the query string `?equipment_broken=photometer` (or anything in the equipment broken field, or other text based relationship) will result in a `ManyRelationValue(pks=[], strings=['photometer'])`. So I think at least we must also handle this case.

Reply:
It is also possible, though maybe not so through the user interface, to have both `pks` and `strings`, if you are entering the URL manually. This is maybe less of a concern but technically it is valid will perform the correct search, for example `categories=4,Assault`